gradle.properties

```diff
@@ -2,6 +2,6 @@ group=org.web3j
 version=4.11.2
 org.gradle.caching=true
 org.gradle.parallel=true
-solidityPluginVersion=0.4.0
+solidityPluginVersion=0.4.1-SNAPSHOT
```

Comment:
revert this change because we don't provide SNAPSHOT versions for gradle plugins
Solidity plugin version will be updated here before releasing new gradle plugin

Reply:
Done.